apps/organizations/models.py

```diff
@@ -320,18 +321,48 @@ def status(self) -> str:
     ##
     # Notifications
 
+    User = get_user_model()
+
+    def _build_notification_message(self, subject: User, verb: str) -> str:
+        """Builds a message that will be displayed as an internal Slack notification.
+        """
+        msg = '{subject_name} ({subject_username}<{email}>) has {verb} an invitation for Organization <{organization_url}|{organization_name}>'.format( # noqa: E501
+            verb=verb,
+            subject_name=subject.profile.get_full_name(),
+            subject_username=subject.username,
+            email=subject.email,
+            organization_url=self.organization.profile_url,
+            organization_name=self.organization.name,
+        )
+        return msg
+
     def build_notification_message__created(self):
         """Builds a message that will be displayed as an internal Slack notification
         when this invitation object is created.
         """
-        msg = '{invited_by_name} ({invited_by_username}) has sent an invitation for organization {organization_name} to {email}'.format(
-            invited_by_name=self.invited_by.profile.get_full_name(),
-            invited_by_username=self.invited_by.username,
-            organization_name=self.organization.name,
```

Comment:
The original (previous) code only displayed `organization.name`, no `organization.profile_url` because the `BaseAbstractionOrganization` (defined here: https://github.com/aweandreverence/django-htk/blob/b469817e147824c1b5dfbc96709e5f6fca56abbf/apps/organizations/models.py#L61) does not have a `profile_url` attribute.